README.md

```diff
@@ -12,7 +12,7 @@ The pillar to give colors and structure to Clay, where is the css and examples o
 
```

Comment:
hey @bryceosterhaus, It would be interesting to add a warning about the v3 version and explain about the branch v2 with components in Metal...

Reply:
Good idea! Ill add that in.

Reply:
Added the reference under the `Clay Components` header. Let me know if you think I should add more to it.

Reply:
Thanks, I would be more "aggressive" with the information, so as not to go unnoticed to someone already know.

I would put something like this below line 3 just as an add-on:
```
🚨This Branch contains the Beta version of Clay v3, if you are looking for the components in Metal.js visit the 2.x branch.
```

That's just a suggestion, but something we should maybe use more, Clay's blog, publish these kinds of decisions we make and the reasons, I think it's important to maintain our relationship with the devs.